.eslintrc.js

```diff
@@ -1,3 +1,5 @@
+// Disabling eslint quote props because prettier clashes with eslint here
+/* eslint-disable quote-props */
```

Comment:
Instead of disabling this with a comment up here, shouldn't we just turn off the configuration value at the bottom of this file? There's a section dedicated to just the lint settings just for this file, so it doesn't make sense to turn on a rule only for this file only to disable it via a comment :)

Reply:
ohhh haha! i did not know